### DISCUSSION TITLE
showing menu only when entering the 1st letter

### DISCUSSION THREAD

**Discussion by crazy-matt** [marlonrichert/zsh-autocomplete#197] [Q&A] [answered]
## What do you want?
I used `zstyle ':autocomplete:tab:*' widget-style menu-select` but it displays the menu even when the command-line is empty. Is it possible to display the menu only when command-line not empty, since the first character is entered.

## Why do you want this?
It would ease in making the terminal+completion features enabled as light as possible visually speaking.

## Who else would benefit from this?
Anyone who wants to trigger the menu on first hit.

## Explain in detail how you would expect this to work
The first part is self explanatory

When I perform the following steps:
1. Command-line is empty. Menu is not displayed, as if there was no completion.
2. I hit anything

Then I expect the following to happen:
* Then the menu popup according to its zstyle settings

**Comment by marlonrichert** [comment, marked as answer]
Duplicate of #195.

**Reply by crazy-matt** [reply]
Hi Marlon,
Sorry for that.
